geniza/corpus/templates/corpus/transcription_export.html

```diff
@@ -0,0 +1,24 @@
+{# minimal html page for transcription export #}
+<html>
+    <head>
+        <title>Transcription of {{ document.title }}</title>
+    </head>
+    <body style="max-width: 860px; margin: auto">
+        <h1>Transcription of {{ document.title }}</h1>
+
+        <p>{{ edition.source.formatted_display|safe }}</p>
+
+        {# include language code when possible #}
+        <section dir="rtl" {% if document.languages.count == 1 and document.languages.first.iso_code %}lang="{{ document.languages.first.iso_code}}"{% endif %}>
+            {% for canvas, html_section in edition.content_html.items %}
+                <div data-canvas="{{ canvas }}">
+                    {# FIXME: why is this lines instead of a block ? #}
```

Comment:
Not sure without testing, but I wonder if this could be similar to the weird dict issues I was having with `iiif_images`… still don't fully understand what Django templates do to dicts

Reply:
django templates call callables (which is usually convenient and sometimes really frustrating); was that problem with a standard dict?

Reply:
It was [this problem](https://github.com/Princeton-CDH/geniza/blob/2ced96efeda7f51d99b78c12956e3cf6d4aa1506/geniza/corpus/templates/corpus/document_detail.html#L23-L28), and IIRC when I tried to use `.values` or `.items` in the template it returned empty strings instead of IIIF image objects, and with `to_list` it started giving me back a string representation of a list…